packages/components/src/internal/components/domainproperties/models.tsx

```diff
@@ -30,18 +30,18 @@ import { getConceptForCode } from '../ontology/actions';
 import { hasPremiumModule } from '../../app/utils';
 
 import {
-    ALL_SAMPLES_DISPLAY_TEXT,
+    ALL_SAMPLES_DISPLAY_TEXT, DECIMAL_RANGE_URI,
```

Comment:
nit: Consider running `npm run lint-branch-fix` prior to final merge.

Reply:
yup, that linting task is in the set of final merge steps Josh is following: 
https://github.com/LabKey/labkey-ui-components/blob/develop/packages/components/docs/localDev.md#merging-changes-into-develop